setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import find_packages, setup
+
+setup(
+    name='delivery_system',
+    version='1.0.0',
+    author="Varvara Minkina",
+    description="Simple version of delivery system",
+    url="https://github.com/Varenique/Delivery_system",
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
```

Comment:
as an option, it is possible to avoid copying the same requirements in `requirements.txt` and in `setup.py`, here is example
https://github.com/heartexlabs/label-studio/blob/master/setup.py

Reply:
also, you can leverage the power of `extras_require` to list your test requirements like:
`extras_require= {'test': ["pytest"]}`

and then `pip install delivery_system[test]` will install test requirements as well

Reply:
fixed